django/db/models/query.py

```diff
@@ -1701,6 +1701,17 @@ def using(self, alias):
         clone._db = alias
         return clone
 
+    def comment(self, message):
+        """Add a comment to the query."""
+        clone = self._clone()
+        if "/*" in message or "*/" in message:
```

Comment:
a regex check for `/\*|\*/` could be slightly faster, avoiding two passes over the string and being more extensible in the future in case we find other syntax that shouldn't be allowed

Reply:
Focusing in only on the point around 'slightly faster' (because whilst I can't think of additional syntax to disallow, that's much of the point around extensibility you bring up ;)). 

IM(limited)E, for any '_sensible_' length string, `x in mystr` (at least where `x` is a constant) is well-optimised, and usually performs better than the equivalent regex, even needing 2 passes, and accounting for compiling the regex beforehand. For the lengths of strings we might _expect_ here, the difference is probably still measured in nanoseconds. If performance were the _only_ consideration (vs. the aforementioned extensibility) it'd be worth checking.